### ISSUE TITLE
XGO build lsp conn error

### ISSUE THREAD

**Issue by mrjosh** [mrjosh/helm-ls#36]
@qvalentin Helllo! Hope you're going fine

Since you change the build tool from gox to xgo
I do get this error while running the `helm_ls_darwin_amd64` building with xgo on `macOS`

```
[ERROR][2023-06-14 05:38:24] .../vim/lsp/rpc.lua:734	"rpc"	"/Users/josh/.bin/helm_ls_darwin_amd64"	"stderr"	'panic: request "initialize" was never replied to\n\ngoroutine 7 [running]:\ngo.lsp.dev/jsonrpc2.ReplyHandler.func1({0x1014fbc28, 0xc000128018}, 0xc000040f40, {0x10ccdc058?, 0xc000056100?})\n\t/go/pkg/mod/go.lsp.dev/jsonrpc2@v0.10.0/handler.go:44 +0x165\ngo.lsp.dev/jsonrpc2.(*conn).run(0xc00047dcc0, {0x1014fbc28, 0xc000128018}, 0xc00027c450)\n\t/go/pkg/mod/go.lsp.dev/jsonrpc2@v0.10.0/conn.go:206 +0x22b\ncreated by go.lsp.dev/jsonrpc2.(*conn).Go\n\t/go/pkg/mod/go.lsp.dev/jsonrpc2@v0.10.0/conn.go:189 +0xb0\n'
```

Can you please double check and see if it's anything wrong with xgo build?
Cause i'm fine while building with `go build -o ./bin/helm_ls`

Thanks

**Comment by qvalentin** [comment]
Hi, I can confirm it is currently also on linux not working. But the problem seems to come from updating the dependencies in this commit: https://github.com/mrjosh/helm-ls/commit/b5b1aef46b6dc67070f916b0b2cffda00b634fcf

`qv@nixos ~/d/g/helm-ls ((4edaec41)|BISECTING)> git bisect bad
Bisecting: 0 revisions left to test after this (roughly 0 steps)
[b5b1aef46b6dc67070f916b0b2cffda00b634fcf] [FIX] removing macOS arm64 due to conflicts with packages that crashes helm_ls`

**Comment by mrjosh** [comment]
The issue was from loading `values.yaml` file while the `WorkspaceFolders.URI` was returning the `file://` string type path.
With the WorkspaceFolders.Name problem is solved [6a1cce9](https://github.com/mrjosh/helm-ls/commit/6a1cce98da3bb1612a45383c5e0547c2370968fd#diff-0097c8e59074fdb549fd2466fb349b2814b208df005f9ccf97b14568d51a5f0aR81)
